src/Testcontainers.Firestore/FirestoreContainer.cs

```diff
@@ -0,0 +1,21 @@
+namespace Testcontainers.Firestore;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public sealed class FirestoreContainer : DockerContainer
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirestoreContainer" /> class.
+    /// </summary>
+    /// <param name="configuration">The container configuration.</param>
+    /// <param name="logger">The logger.</param>
+    public FirestoreContainer(FirestoreConfiguration configuration, ILogger logger)
+        : base(configuration, logger)
+    {
+    }
+
+    public void SetEmulatorHost()
+    {
+        Environment.SetEnvironmentVariable("FIRESTORE_EMULATOR_HOST", $"{Hostname}:8080/");
```

Comment:
As previously requested (in the other pull request), can we please not rely on the environment variable to create a client connection? Environment variables are shared (min. across the process) and prevents developers from running multiple instances concurrently.

Reply:
done